source/adminguide/virtual_machines.rst

```diff
@@ -416,7 +416,7 @@ To delete an affinity group:
 Virtual Machine Snapshots
 -------------------------
 
-(Supported on VMware and XenServer)
+(Supported on VMware, XenServer and KVM partially)
```

Comment:
if it's only with NFS may be we can reword to `KVM (only NFS)`. Makes it more informative heading.

Reply:
fixed.